mail/views_test.py

```diff
@@ -140,43 +146,106 @@ def test_automatic_email(self):
         email_results = ['a@example.com', 'b@example.com']
         request_data = self.request_data.copy()
         request_data['send_automatic_emails'] = True
-
-        assert AutomaticEmail.objects.count() == 0
-        search_dict = create_search_obj(
+        for email in email_results:
```

Comment:
Can you split these two automatic email test cases into a separate test class? You're performing the same ~20 lines of data creation for both of the cases